packages/rest/src/rest.server.ts

```diff
@@ -599,7 +599,9 @@ export class RestServer extends Context implements Server, HttpServerLike {
    */
   async stop() {
     // Kill the server instance.
-    await this._httpServer.stop();
+    if (this._httpServer) {
```

Comment:
We usually use reverse logic + early return.

```ts
if (!this._httpServer) return;
await this._httpServer.stop();
```

I think you should also unset the http server after it was stopped, so that subsequent calls of `.stop()` don't try to stop an already stopped server. Thoughts?

```ts
if (!this._httpServer) return;
await this._httpServer.stop();
this._httpServer = undefined;
```

Reply:
@bajtos As for early return style, shall I refactor the http server stop method too? currently it's like this

 ```js
public async stop() {
    if (this.httpServer) {
      this.httpServer.close();
      await pEvent(this.httpServer, 'close');
      this._started = false;
    }
  } 
```